frontend/src/text_resources.js

```diff
@@ -9,7 +9,7 @@ let LOCALIZE = new LocalizedStrings({
       dashboardTabTitle: "Dashboard",
       sampleTest: "Sample eMIB",
       statusTabTitle: "Status",
-      psc: "Public Service Commission",
+      psc: "Public Service Commission of Canada",
```

Comment:
Do we have a translation for the French?

Reply:
cc @siobhanozege

Reply:
@MichaelCherry update the FR to: "Commission de la fonction publique du Canada"

Reply:
The french already had that value